force-app/main/default/classes/VariablesDataTypesOperators.cls

```diff
@@ -242,14 +272,14 @@ public with sharing class VariablesDataTypesOperators {
         final Double SUBTRACT_FACTOR = 32.0;
         final Double MULTIPLY_FACTOR = 5.0;
         final Double DIVIDE_FACTOR = 9.0;
-    
-        return null; // Replace null with the variable you used to store the result
+    Double fahrenheitToCelsius = (fahrenheit - SUBTRACT_FACTOR) * MULTIPLY_FACTOR/DIVIDE_FACTOR;
+        return fahrenheitToCelsius; // Replace null with the variable you used to store the result
     }
 
     
 
     /**
-     * Question 17
+     * Question 17  What does cast mean? <---Correct!! BW
```

Comment:
Casting  in this context means that you have one datatype and want to convert it to another 

Double myDouble = 45.67;

Integer myInt = (Integer) myDouble;   //casting the double to an Integer

System.debug('The integer value is: ' + myInt);  //45

Reply:
Ah, Thank you!